app/test_main.py

```diff
@@ -1,3 +1,36 @@
+import pytest
+
 from app.main import get_human_age
 
-# write your code here
+
+test_cases = [
+    (0, 0, [0, 0]),
+    (14, 14, [0, 0]),
+    (15, 15, [1, 1]),
+    (23, 23, [1, 1]),
+    (24, 24, [2, 2]),
+    (27, 27, [2, 2]),
+    (28, 28, [3, 2]),
+    (100, 100, [21, 17])
+]
+
+
+@pytest.mark.parametrize("cat_age, dog_age, expected_result", test_cases)
+def test_get_human_age(cat_age: int,
+                       dog_age: int,
+                       expected_result: list) -> None:
+    assert get_human_age(cat_age, dog_age) == expected_result
+
+
+def test_no_one_or_all_values() -> None:
+    try:
+        assert get_human_age() == [0, 0]
+    except TypeError:
+        print("function take 2 parameters")
```

Comment:
consider to use pytest.raises for handling exceptions in unit tests instead of using a try...except block because it provides more concise and explicit handling of expected exceptions